hardware/Netatmo.cpp

```diff
@@ -2561,7 +2585,9 @@ bool CNetatmo::ParseHomeStatus(const std::string& sResult, Json::Value& root, st
 						b << ";";
 						b << boiler_boost;
 						std::string sValue = b.str().c_str();
+						const double Level = module["boiler_valve_comfort_boost"].asDouble();
```

Comment:
@JurgenLB , add a variable and value that is not used? Or was, but now commented out.. remove as well?

Reply:
this belongs to the //SendSwitch

Reply:
See above. Adding a variable and not use it does not make things clearer.